Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/Callout.qml

```diff
@@ -313,12 +318,10 @@ Item {
 
         // once leader position is finalized
         if (findBestLeaderPosition(anchorPointx, anchorPointy)) {
-            if (platform == "ios")
-                adjustRelativePositionOfCanvasFrame(anchorPointx, anchorPointy, calloutLayout.width - 2*cornerRadius, calloutLayout.height - cornerRadius);
+            if (platform === "ios")
```

Comment:
@nandinirao no, I tried removing it and tweaking it, but it did not make it look better on iOS. So, I thought it would be best to leave it as is for now